docs/make_work.jl

```diff
@@ -170,18 +180,22 @@ function doit(
   end
 
   cd(joinpath(Oscar.oscardir, "docs")) do
-    DocMeta.setdocmeta!(Oscar, :DocTestSetup, :(using Oscar; Oscar.AbstractAlgebra.set_current_module(@__MODULE__)); recursive=true)
+    DocMeta.setdocmeta!(Oscar, :DocTestSetup, doctestsetup(); recursive=true)
     DocMeta.setdocmeta!(Oscar.Hecke, :DocTestSetup, :(using Hecke; import AbstractAlgebra; AbstractAlgebra.set_current_module(@__MODULE__)); recursive=true)
     DocMeta.setdocmeta!(Oscar.AbstractAlgebra, :DocTestSetup, :(using AbstractAlgebra; AbstractAlgebra.set_current_module(@__MODULE__)); recursive=true)
     DocMeta.setdocmeta!(Oscar.Nemo, :DocTestSetup, :(using Nemo; import AbstractAlgebra; AbstractAlgebra.set_current_module(@__MODULE__)); recursive=true)
 
+    if !(doctest isa Bool && doctest == false)
```

Comment:
Wouldn't this also work?
```suggestion
    if doctest !== false
```

Reply:
Ah, much better, yes